resources/public/api/conf/1.0/other_reliefs_amend.yaml

```diff
@@ -17,12 +17,16 @@ description: |
       <tbody>
           <tr>
               <td><p>N/A - DEFAULT</p></td>
-              <td><p>Simulate success response.</p></td>
+              <td><p>Simulates a success response.</p></td>
           </tr>
           <tr>
                <td><p>SUBMISSION_FAILED</p></td>
                <td><p>Simulates the scenario where a submission has failed.</p></td>
           </tr>
+          <tr>
+               <td><p>STATEFUL</p></td>
+               <td><p>Simulates a stateful success response.</p></td>
```

Comment:
Relief investments:
the stateful description needs to be updated to the newer version!